src/pages/UpSertUser/UpSertPage/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState } from "react";
+
+import { Maybe, Users } from "src/generated/graphql";
+
+import BoxBase from "src/components/Boxs/BoxBase";
+import ButtonBase from "src/components/Buttons/ButtonBase";
+import ImageBase from "src/components/Images/ImageBase";
+import TextFieldBase from "src/components/Textfields/TextFieldBase";
+import TypographyBase from "src/components/Typography/TypographyBase";
+import { anonymousAvatarLink, IUser } from "src/pages/Users/components/UserCard";
+
+type UserByPk = Maybe<
```

Comment:
Why write this type again when you can just import from generated types?

Reply:
![image](https://user-images.githubusercontent.com/58205909/133928796-023b19fd-8c78-4a78-8254-963c0670d148.png)

Reply:
my bad